utils.jl

```diff
@@ -4,21 +4,33 @@ using Dates
     {{meta}}
 
 Plug in specific meta information for a blog page. The `meta` local page
-variable should be given as a list of tuples of pairs like so:
+variable should be given as an iterable of 3-tuples like so:
 ```
-@def meta = [("property"=>"og:video", "content"=>"http://example.com/"),
-             ("propery"=>"og:title", "content"=>"The Rock")]
+@def meta = [("property", "og:video", "http://example.com/"),
+             ("name", "twitter:player", "https://www.youtube.com/embed/XXXXXX")]
 ```
+A full example can be found in `blog/2020/05/rr.md`.
 """
 function hfun_meta()
-    m = locvar(:meta)
-    io = IOBuffer()
-    for tuple in locvar(:meta)
-        write(io, "<meta ")
-        for (prop, val) in tuple
-            write(io, "$prop=\"$val\" ")
+    p = "property"
+    # default og properties, can be overwritten by the user
+    ogdflt = (
+        title = (p, "og:title", "The Julia Language"),
```

Comment:
On Blog pages, can we default this to the post title, e.g. here's what it currently looks like on facebook:

<img width="588" alt="Screen Shot 2020-09-25 at 7 17 15 AM" src="https://user-images.githubusercontent.com/1291671/94261121-56869e80-feff-11ea-8e38-7361aaf2d94b.png">

Rather than the "The Julia Language" can we do the post title for og:title if we're on the blog?

Reply:
yes, I'll open another PR with that